ios/FluentUI/Table View/TableViewCellTokenSet.swift

```diff
@@ -213,9 +213,12 @@ extension TableViewCellTokenSet {
     /// The leading margin for the labelAccessoryView.
     static let labelAccessoryViewMarginLeading: CGFloat = GlobalTokens.spacing(.size80)
 
-    /// The trailing margin for the labelAccessoryView.
+    /// The trailing margin for the labelAccessoryView of the title label.
     static let labelAccessoryViewMarginTrailing: CGFloat = GlobalTokens.spacing(.size80)
 
+    /// The trailing margin for the labelAccessoryView of the subtitle label.
+    static let sublabelAccessoryViewMarginTrailing: CGFloat = GlobalTokens.spacing(.size40)
```

Comment:
i think we should match the name of the actual UILabel. ex) subtitleLabelAccessoryViewMarginTraining and titleLableAccessoryviewMarginTrailing for easy search. which is what subtitletwolineheight did as well.